packages/experiments/src/components/Button/Button.types.tsx

```diff
@@ -9,16 +10,21 @@ export type IButtonStates = 'baseState' | 'enabled' | 'disabled' | 'expanded';
 
 export type IButtonVariants = 'baseVariant' | 'primary' | 'circular';
 
-export type IButtonSlots = 'root' | 'stack' | 'text' | 'icon' | 'menuIcon';
+export interface IButtonSlots {
+  root?: IHTMLButtonSlot;
+  stack?: IHorizontalStackSlot;
```

Comment:
Is there a plan to write JSDoc in a subsequent PR?

Reply:
Man I just noticed the entire file has 0 JSDoc :| I guess the documentation writing will be done before promotion to OUFR. @dzearing

Reply:
Slots is an experiment and Button will still change quite a bit before promotion.

Reply:
@cliffkoh The props at this point are not set in stone at all, and I intentionally did not spin a lot of effort in making this perfect, until they have been finalized and reviewed.